packages/augur-ui/src/modules/create-market/components/review.tsx

```diff
@@ -30,7 +39,90 @@ export default class Review extends React.Component<
 
```

Comment:
similar to the above comment, we should not keep repeating the `newMarket` part, deconstruct the newMarket object and then just reference `category` or `tag1` etc